mon-pix/app/components/feedback-panel.js

```diff
@@ -1,138 +1,107 @@
+import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
 import { isEmpty } from '@ember/utils';
-import buttonStatusTypes from 'mon-pix/utils/button-status-types';
-import Component from '@ember/component';
-import classic from 'ember-classic-decorator';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
 
+import buttonStatusTypes from 'mon-pix/utils/button-status-types';
 import { topLevelLabels, questions } from 'mon-pix/static-data/feedback-panel-issue-labels';
 
-@classic
-@classNames('feedback-panel')
 export default class FeedbackPanel extends Component {
   @service store;
 
-  assessment = null;
-  challenge = null;
-
-  displayQuestionDropdown = false;
-  displayTextBox = null;
-  emptyTextBoxMessageError = null;
-  nextCategory = null;
-  isFormOpened = false;
-  quickHelpInstructions = null;
-  sendButtonStatus = buttonStatusTypes.unrecorded;
+  @tracked content = null;
+  @tracked displayQuestionDropdown = false;
+  @tracked displayTextBox = false;
+  @tracked emptyTextBoxMessageError = null;
+  @tracked isFormOpened = false;
+  @tracked isFormSubmitted = false;
+  @tracked nextCategory = null;
+  @tracked quickHelpInstructions = null;
 
   _category = null;
-  _content = null;
-  _isSubmitted = false;
   _questions = questions;
+  _sendButtonStatus = buttonStatusTypes.unrecorded;
+
+  constructor(owner, args) {
+    super(owner, args);
+    this._resetPanel();
+  }
 
-  @computed('context')
   get categories() {
-    const context = this.context === 'comparison-window' ? 'displayOnlyOnChallengePage' : 'displayOnlyOnComparisonWindow';
+    const context = this._isComparisonWindowContext ? 'displayOnlyOnChallengePage' : 'displayOnlyOnComparisonWindow';
     return topLevelLabels.filter((label) => !label[context]);
   }
 
   get isSendButtonDisabled() {
-    return this.sendButtonStatus === buttonStatusTypes.pending;
-  }
-
-  _resetPanel() {
-    this.set('_isSubmitted', false);
-    this.set('emptyTextBoxMessageError', null);
-  }
-
-  didReceiveAttrs() {
-    super.didReceiveAttrs();
-    this._resetPanel();
-    this.set('_content', null);
+    return this._sendButtonStatus === buttonStatusTypes.pending;
   }
 
-  _showFeedbackActionBasedOnCategoryType(category) {
-    this.set('displayTextBox', false);
-    this.set('quickHelpInstructions', null);
-
-    if (category.type === 'tutorial') {
-      this.set('quickHelpInstructions', category.content);
-    } else if (category.type === 'textbox') {
-      this.set('displayTextBox', true);
-    }
-  }
-
-  _scrollIntoFeedbackPanel() {
-    const feedbackPanelElements = document.getElementsByClassName('feedback-panel__view');
-    if (feedbackPanelElements && feedbackPanelElements[0]) {
-      feedbackPanelElements[0].scrollIntoView();
+  get isToggleFeedbackFormDisabled() {
+    if (this._isComparisonWindowContext) {
+      return true;
     }
+    return false;
```

Comment:
A priori, on doit pouvoir simplifier le contenu de cette méthode comme suit :
```suggestion
  get isToggleFeedbackFormDisabled() {
    return this._isComparisonWindowContext
```